spec/models/chess_piece_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe ChessPiece do
+  describe "#horizontal_move?" do
+    context "given a piece at [4, 4]" do
+      let(:piece) {ChessPiece.create(position_x: 4, position_y: 4)}
+      it "will return true when trying to move to [5,4]" do
+        expect(piece.horizontal_move?(5,4)).to be_truthy
+        expect(piece.horizontal_move?(4,5)).to be_falsey
```

Comment:
Sorry I'm going to be real stickler for this because i don't want bad habits to form.

Can you separate the `truthy / falsey` statements? I.e. move this line into it's own `it` block:

```
it "will return false when trying to move to [4, 5]" do
  expect(piece.horizontal_move?(4,5)).to be_falsey
end
```

**reason**: because when this test fails, it's easier to see which `case` is causing the error ( truthy / falsey ).

**Another Option**:

Change the wording of the `it` description on line 7 to "will return true or false based on coordinates"
and then you would change your `expects` to look like:
```
expect(piece.horizontal_move?(5,4)). to be_truthy, "Moving to (5, 4) returned false"
expect(piece.horizontal_move?(5,4)). to be_falsey, "Moving to (5, 5) returned true"
```

To make the errors clearer.